app/views/local_transaction/results.html.erb

```diff
@@ -48,14 +50,14 @@
   <% end %>
   <div class="search-again">
     <%= render "govuk_publishing_components/components/back_link", {
-      href: local_transaction_search_path(@publication.slug)
+      href: local_transaction_search_path(@publication.slug),
     } %>
   </div>
   <section class="more">
     <div class="more">
-      <%= render "govuk_publishing_components/components/govspeak", {
-        content: sanitize(@publication.more_information)
-      } %>
+      <%= render "govuk_publishing_components/components/govspeak", {} do %>
```

Comment:
Is the `{}` needed?

Reply:
Nope, it's not needed. It renders the same with or without the `{}` - I left it in as that's what the examples in the component gem docs do.